examples/fscloud/main.tf

```diff
@@ -84,12 +84,12 @@ module "cbr_account_level" {
 
   # Demonstrates how additional context to the rules created by this module can be added.
   # This example open up:
-  #   1. Flows from icd mongodb, postgresql to kms on private endpoint
+  #   1. Flows from icd mongodb, postgresql to KMS and HPCS on private endpoint
   #   2. Flow from schematics on public kms endpoint
   #   3. Add a block of ips to schematics public endpoint
   #   4. Flow from vpc(s) specified in input zone_vpc_crn_list to postgresql private endpoint
-  custom_rule_contexts_by_service = {
-    "kms" = [{
+  custom_rule_contexts_by_service = merge({
+    for key in var.kms : key => [{
```

Comment:
Is adding those rules in the example still relevant given that the root module add rules for all icd service now?

Reply:
Make sense, removed the rule for ICD.